### DISCUSSION TITLE
Rust ORM -> Raw SQL String

### DISCUSSION THREAD

**Discussion by hariria** [Brendonovich/prisma-client-rust#247] [Ideas]
I'm not sure if this is already possible but hypothetically could there be a method that converts a prisma ORM query into a raw sql string?

I'm imagining something like

```rust
client
    .users()
    .create(guid.to_string(), "".to_string(), vec![])
    .toString()
```

would return a string like so
```sql
INSERT INTO users (id, name) VALUES ("guid", "name");
```

**Comment by Brendonovich** [comment]
Prisma's engines don't provide a way to do this, so unfortunately I won't be able to either unless https://github.com/prisma/prisma/issues/5052 is implemented.

**Reply by hariria** [reply]
Got it thanks!
